tutorials/simulators/5_noise_transformation.ipynb

```diff
@@ -57,7 +57,9 @@
     "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    results = approximate_quantum_error(Kraus([K0, K1]), operator_string="reset")
</pre><p>I am not familiar with this function in Aer. Can you provide a bit of context for this change? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
This change is because `approximate_quantum_error` changed the acceptable type of the first argument to be `QuantumError` or `QuantumChannel`.  Without this change, this line will raise
```
DeprecationWarning: Support of types other than QuantumError or QuantumChannel for error to be approximated has been deprecated as of qiskit-aer 0.10.0 and will be removed no earlier than 3 months from that release date.
  results = approximate_quantum_error([K0, K1], operator_string="reset")
```